### PR TITLE
Modify appliance-multi-hosts to work on eco vsphere

### PR DESCRIPTION
**Pull request by shellymiron**
Modified appliance-multi-hosts to work on eco vsphere.
This playbook configures the NTP settings on a vCenter server using the `vmware.vmware_rest.appliance_ntp` module with delegated execution on localhost

### CODE REVIEW

File: tests/integration/targets/vmware_rest_appliance-multi-hosts/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+- hosts: localhost
+  gather_facts: no
+
+  tasks:
+    - name: Import eco-vcenter credentials
```
Comment (anna-savina) [inline tests/integration/targets/vmware_rest_appliance-multi-hosts/playbook.yml:5]: you don't need to import it, unless you use vars from this file in a test. Authorization occurs in init-eco.sh

File: tests/integration/targets/vmware_rest_appliance-multi-hosts/tasks/appliance_ntp.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+- name: Manage NTP Configuration on vCenter
+  block:
+    - name: Reset the NTP configuration
```
Comment (anna-savina) [inline tests/integration/targets/vmware_rest_appliance-multi-hosts/tasks/appliance_ntp.yml:4]: Should NTP configuration be restored at the end of the test?

Comment (shellymiron) [reply tests/integration/targets/vmware_rest_appliance-multi-hosts/tasks/appliance_ntp.yml:4]: I thought about that, but I don't recall the primary NTP that was configured before this test lol
But anyway, it was probably something like "time.vmware.com", and the last one to be configured is "time.google.com", so I think google is better?

File: tests/integration/targets/vmware_rest_appliance-multi-hosts/playbook.yml (hunk 1 above)
Comment (shellymiron) [reply tests/integration/targets/vmware_rest_appliance-multi-hosts/playbook.yml]: fixed :)

**Review by anna-savina — APPROVED** [review]
LGTM

**Comment by prabinovRedhat** [comment]
lgtm

**Review by prabinovRedhat — APPROVED** [review]

**Comment by mikemorency** [comment]
recheck

**Comment by mikemorency** [comment]
recheck

**Pull request merged**